src/exercises/exercise05/Link.md

```diff
@@ -1,4 +1,16 @@
+Normal link:
+
 ```jsx
-<Link href="/">Normal link</Link>{' '}
-<Link href="/" shy>Shy link</Link>
+<Link href="/">Normal link</Link>
+```
+
+Shy link. Use for secondary content like footers or small copy.
+
+```jsx
+<p>
```

Comment:
Should we use `Text` here instead? We're teaching not to use HTML elements :)

Reply:
They don't share any styles now — link only has color ;-)

Reply:
Not sure I understand what you're saying. Is Text implemented at this point?

Reply:
Final implementations don't share styles:

https://github.com/sapegin/component-driven-development/blob/master/src/components/core/Link.js
https://github.com/sapegin/component-driven-development/blob/master/src/components/core/Text.js

Reply:
But maybe they should, then we could use a Link outside Text, for example in navigation.

Reply:
Done.